### PR TITLE
creating a dev branch with a readme file

### PR DESCRIPTION
**Pull request by Roberamelaek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Creating a Dev branch and pushing it to git hub
```
Comment (kirubeltadesse) [inline README.md]: Could you include a description about what the repo is used and what you plan to do in the future?

[Here](https://www.markdownguide.org/basic-syntax/) is a good resource on how to format a markdown file.

**Review by kirubeltadesse — COMMENTED** [review]
Awesome 👍🏼 you just created your first PR ever 🎉 !

Please see the comment I left you and update and re-push your changes.

File: README.md (hunk 1 above)
Comment (mainhubn) [reply README.md]: ok

**Comment by Roberamelaek** [comment]
the read me is done

**Review by kirubeltadesse — COMMENTED** [review]
The contain looks good except one more thing
What you used here is `html` while this also work but the file we are using is a `markdown` file can use use the markdown highlighting instated?

**Review by kirubeltadesse — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**